src/VolleyManagement.WebClient/spa/app/Services/tokenInterceptor.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { HttpRequest, HttpHandler, HttpEvent, HttpInterceptor, HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/do';
+
+@Injectable()
+export class TokenInterceptor implements HttpInterceptor {
```

Comment:
What is purpose of this?

Reply:
Intercept an outgoing HttpRequest and optionally transform it or the response.
I use it for adding { withCredentials: true } to request.